### PR TITLE
Update LocalSettings.php

### PR DESCRIPTION
**Pull request by 1-2-1-2-3-Miraheze**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3146,8 +3147,12 @@
 			),
 		),
 		'+wikicanadawiki' => array(
-			'Founder' => array(
+			'founder' => array(
 				'read' => true,
+				'checkuser' => true,
```
Comment (Reception123) [inline LocalSettings.php]: Unfortunately, we currently don't give these permissions to users on wikis, they are only provided for stewards.

**Comment by NDKilla** [comment]
This shouldn't be merged in if there are users in the 'Founder' group on the wiki. I can't check because I can't login on my phone.

**Comment by NDKilla** [comment]
@reception123 as a sysadmin you can change the user's rights on wiki before renaming if you need

**Comment by Reception123** [comment]
@NDKilla I can't login so I can't check :( I can't merge until the checkuser and non-authorized rights are changed anyways

File: LocalSettings.php (hunk 1 above)
Comment (Reception123) [inline LocalSettings.php]: Therefore you need to remove "checkuser", "checkuser-log", "renameuser" and "usermerge" and then I can merge this PR for changing the founder permissions

Comment (1-2-1-2-3-Miraheze) [inline LocalSettings.php]: The only user in the "Founder" group is my sister

Comment (LegoFan4000) [inline LocalSettings.php]: Sorry we cannot give out those permissions

File: LocalSettings.php [hunk 2]
```diff
@@ -3146,8 +3147,12 @@
...
+				'checkuser-log' => true,
+				'renameuser' => true,
+				'usermerge' => true,
```
Comment (LegoFan4000) [inline LocalSettings.php]: Please remove this

File: LocalSettings.php (hunk 1 above)
Comment (1-2-1-2-3-Miraheze) [inline LocalSettings.php]: So what would happen if you merged this as is? Would it crash the DB again?

File: LocalSettings.php [hunk 3]
```diff
@@ -3146,8 +3147,12 @@
 			),
 		),
 		'+wikicanadawiki' => array(
-			'Founder' => array(
+			'founder' => array(
 				'read' => true,
+				'checkuser' => true,
+				'checkuser-log' => true,
```
Comment (LegoFan4000) [inline LocalSettings.php]: Please remove this

File: LocalSettings.php (hunk 1 above)
Comment (Reception123) [inline LocalSettings.php]: The founder thing is fine, the CheckUser,  etc.  permissions need to be removed

File: LocalSettings.php [hunk 4]
```diff
@@ -3146,8 +3147,12 @@
...
+				'renameuser' => true,
```
Comment (LegoFan4000) [inline LocalSettings.php]: Please remove this

File: LocalSettings.php (hunk 1 above)
Comment (Reception123) [inline LocalSettings.php]: @1-2-1-2-3-Miraheze No, this is not related to the DB. We simply don't give out these permissions for privacy reasons

Comment (1-2-1-2-3-Miraheze) [inline LocalSettings.php]: Why? Is there some sort of technical/database restrictions that prevent non-Stewards from using theses permissions?

**Comment by Reception123** [comment]
@Legofan4000 Please don't add useless comments, I already commented and pointed it out
